src/database/data/accommodation-categories.data.ts

```diff
@@ -1,4 +1,4 @@
-import { AccommodationCategory } from 'src/modules/accommodations/entities';
+import { AccommodationCategory } from 'src/modules/categories/entities/category.entity';
```

Comment:
Could you create index file in entities for better import/export?

Reply:
done